utils/create-database.js

```diff
@@ -0,0 +1,63 @@
+// ```js
+// // utils/create-database.js
+
+const mysql = require('mysql2/promise');
+
+// require path to handle file paths
+const path = require('path');
+
+// extract any command line arguments from argv
+const args = process.argv.slice(2)[0];
+
+// use args to determine if .env or .env.test should be loaded
+const envFile = args === 'test' ? '../.env.test' : '../.env';
+
+// load environment variables from env files
+require('dotenv').config({
+  path: path.join(__dirname, envFile),
+});
+
+// destructure environment variables from process.env
```

Comment:
This is self explanatory and doesnt really warrant a comment, as are lines 26, 33, 44, 62 & 63. Generally comments are only for explaining something that either would not be obvious in a short time or to provide context (EG _why_ something is done the way it is and not _how_)